examples/tutorial/jupyter/execution/omnisci_on_native/local/exercise_3.ipynb

```diff
@@ -0,0 +1,139 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![LOGO](../../../img/MODIN_ver2_hrz.png)\n",
+    "\n",
+    "<center><h2>Scale your pandas workflows by changing one line of code</h2>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 3: Not Implemented\n",
+    "\n",
+    "**GOAL**: Learn what happens when a function is not yet supported in Modin."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When functionality has not yet been implemented for `OmnisciOnNative` execution, we default to pandas as follows\n",
+    "\n",
+    "![](../../../img/omnisci_convert_to_pandas.png)\n",
+    "\n",
+    "We convert the Modin dataframe to a pyarrow.Table, perform a lazy tree execution in OmniSci, render it as a pyarrow.Table, convert it to pandas to perform the operation, and then convert it back to Modin when complete. These operations will have a large overhead due to the communication involved and will take longer than pandas.\n",
+    "\n",
+    "When this is happening, a warning will be given to the user to inform them that this operation will take longer than usual. For example, `DataFrame.apply` is not supported. In this case, when a user tries to use it, they will see this warning:\n",
+	"\n",
+	"```\n",
+	"UserWarning: `DataFrame.apply` defaulting to pandas implementation.\n",
+	"```"
```

Comment:
```suggestion
	"```\n",
    "\n",
    "#### Relation engine limitations\n",
    "As the `OmnisciOnNative` execution is backed by relation algebra based DB, there is a certain set of limitations on operations that could be used in Modin with such execution. For example arbitary functions in `DataFrame.apply` are not supported as the OmniSci engine can't execute python callables againts its tables, this means that `DataFrame.apply(python_callable)` will **always** be defaulting to pandas. \n",
    "\n",
    "For more info about `OmnisciOnNative` limitations visit the appropriate section on read-the-docs: [relation algebra limitations](https://modin.readthedocs.io/en/stable/flow/modin/experimental/core/execution/native/implementations/omnisci_on_native/index.html#relational-engine-limitations).\n",
    "\n",
    "If your flow mainly operates with non-relational algebra operations, you should better choose non-OmniSci execution (i.e. `PandasOnRay`)."
```